CryptKeeperZK/crypt-keeper-extension
# feat: disconnect identity

## Explanation

This PR adds disconnect action for identity home list.

Details are below:
- [x] Add disconnect identity action for home list
- [x] Add ready message for extension initialization
- [x] Get rid of jest deprecated methods

## Related Issues

Closes #933 

## Screenshots

<details>
<summary>Expand</summary>

![image](https://github.com/CryptKeeperZK/crypt-keeper-extension/assets/14254374/9c0ba32a-11d5-4f98-a8d7-87a67e8a6446)

</details>

## Manual Testing Steps

1. Try to connect identity
2. Then disconnect it
3. Check disconnect option is not available for identities without connected origin
4. Check extension doesn't throw error related to #933 

## Pre-Merge Checklist

### Assignees Checklist


### Reviewers Checklist

- [ ] Manual testing checked and passed.
